code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -327,7 +327,7 @@
 			FindTarget()
 
 
-/mob/living/simple_animal/hostile/proc/AttackingTarget()
+/mob/living/simple_animal/hostile/proc/AttackingTarget(atom/A)
```

Comment:
Is this something you can atomize?

Reply:
I don't think its that massive of a change and it isn't really it's own thing, it's an addition specific to this feature.

Reply:
I'd merge passing the variable on its own.